Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9-alpine
+
+RUN touch /config.json
+
+RUN adduser --disabled-password --home=/app syncmymoodle
+COPY --chown=syncmymoodle:syncmymoodle . /app
+
+USER syncmymoodle
+WORKDIR /app
+
+RUN pip install -r requirements.txt
```

Comment:
Preferably the requirements should be copied alone and installed before the rest of the application code to utilize docker layer caching